src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -59,24 +58,13 @@
     public static final String PREAMBLE_WHITESPACE = "\t  \r  \n";
     public static final String PREAMBLE_NON_EMPTY = "NonEmptyPreamble";
 
-    public static final EditCommand.EditPersonDescriptor DESC_AMY;
-    public static final EditCommand.EditPersonDescriptor DESC_BOB;
-
     // for logic in sochedule
     public static final String VALID_TASK_NAME = "Sample Task";
     public static final String VALID_TASK_DEADLINE = "2022-01-01";
     public static final String VALID_PRIORITY = "5";
     public static final String TASK_NAME_SAMPLE = " " + PREFIX_NAME + VALID_TASK_NAME;
     public static final String TASK_DEADLINE_SAMPLE = " " + PREFIX_DEADLINE + VALID_TASK_DEADLINE;
     public static final String PRIORITY_SAMPLE = " " + PREFIX_PRIORITY + VALID_PRIORITY;
-    static {
```

Comment:
this may not be an issue but I am thinking maybe we will need something similar for our tests later